README_NEW.md

```diff
@@ -30,28 +30,23 @@ For example, PySyft can improve your product based on customers' private data wi
 
 
 <h5 align="center">
-    <img src="packages/syft/docs/img/Working_infographic.gif" alt="working_infographic" width="400">
+    <img src="packages/syft/docs/img/Working_infographic.gif" alt="working_infographic" width="1080">
 </h5>
  
-
-
-Data Owners provide (digital) datasets which they would like to make available for study by an outside party they may or may not fully trust has good intentions.
-
-Domain server manages the remote study of the data by a Data Scientist and allows the Data Owner to handle the data and incoming requests from the Data Scientist. It also acts as a gatekeeper for the Data Scientist's access to the data to compute and experiment with the results.
-
-Network server provides services to the network of Data Owners and Data Scientists, such as dataset search and bulk project approval (legal/technical) to participate in a project. It acts as a bridge between the members (Domains) and its subscribers (End Users) and can provide access to a collection of domains at once.
-
-Data Scientists are end users who desire to perform computations or answer a specific question using data owned by someone else. They can see a list of datasets hosted on the Domains and can request to get the results.
+### Important General Terms
+* **Data Owners** provide datasets which they would like to make available for study by an outside party they may or may not fully trust has good intentions.
+* **Domain server** manages the remote study of the data by a Data Scientist and allows the Data Owner to handle the data and incoming requests from the Data Scientist. It also acts as a gatekeeper for the Data Scientist's access to the data to compute and experiment with the results.
+* **Network server** provides services to the network of Data Owners and Data Scientists, such as dataset search and bulk project approval (legal/technical) to participate in a project. It acts as a bridge between the members (Domains) and its subscribers (End Users) and can provide access to a collection of domains at once.
+* **Data Scientists** are end users who desire to perform computations or answer a specific question using data owned by someone else. They can see a list of datasets hosted on the Domains and can request to get the results.
 
 The steps performed by the respective personas are shown below:
 
 
 <h5 align="center"> 
-    <img src="packages/syft/docs/img/big-picture.png" alt="big-picture-overview" width="400">
+    <img src="packages/syft/docs/img/big-picture.png" alt="big-picture-overview" width="720">
 </h5>
 
 
-
 # Deployment
```

Comment:
@madhavajay, I needed your input on the Deployment section. Right now, the deployment section is not the priority. Hence, we were thinking of moving everything from the current README into separate files and giving links to them here. Later, we can make separate tutorials for each deployment method and have them more organized. Let me know what you think about this.

Also, the above quickstart section is being worked on by @shubham, and once that part is added, the text part above will give more context. Right now, it looks only text-text, which I believe was not our intention, but after discussing it with @irina, we think the part makes sense to explain the infographic. 

However, let us know, and we can make changes anytime. 

Thank you :)